### ISSUE TITLE
Some thoughts based on the readme 

### ISSUE THREAD

**Issue by ghost** [JosiahSwaim/Punch-IN-OUT#1]
Hi,

Got here from reddit. Like the program logic you wrote. I do have 1 thing though.

Two barcodes seems Clunky and will be a little error prone.

How about simplifying the process and have the barcode read only a name.

The script can then check the file and see if John Doe is in or not if yes and it's longer than time set to out if no set to in.

For scan that is. The rest logic remains. This is just a thought I had after reading the readme.

Another thing maybe consider using sqlite ?

**Comment by brenttaylor** [comment]
I would second the use of SQLite.  You've got a lot more file i/o than is really necessary.

**Comment by josai** [comment]
Yeah it's probably a better idea to use SQL but there's never going to be more than 40 people using it on any given day so it's not a priority at this point. There's usually at least several seconds if not minutes between people scanning in so the lack of speed isn't a big deal.

And as far as having a single barcode instead of the two, people often will accidentally double scan themselves or forget if they scanned in/out. And also the people scanning in can't see the computer monitor so there's no way for them to know if they are in/out. So if we make it so they flip the id card to show the reverse barcode then they always know if they are checking in or out.

This way it's maybe a little more hassle for the users but it's more accurate in determining whose actually in the building at any given moment, which is the whole point of the system when it was first created but as a paper binder.
